tests/store/model_registry/test_file_store.py

```diff
@@ -1512,3 +1512,69 @@ def predict(self, context, model_input, params=None):
         mv2 = store.search_model_versions("name = 'model2'", max_results=10)
         assert len(mv2) == 1
         assert mv2[0].name == "model2"
+
+
+def test_copy_model_version(store):
+    name1 = "test_for_copy_MV1"
+    store.create_registered_model(name1)
+    src_tags = [
+        ModelVersionTag("key", "value"),
+        ModelVersionTag("anotherKey", "some other value"),
+    ]
+    with mock.patch("time.time", return_value=456778):
+        src_mv = _create_model_version(
+            store, name1, tags=src_tags, run_link="dummylink", description="test description"
+        )
+
+    # Make some changes to the src MV that won't be copied over
+    store.transition_model_version_stage(
+        name1, src_mv.version, "Production", archive_existing_versions=False
+    )
+
+    name2 = "test_for_copy_MV2"
+    with mock.patch("time.time", return_value=456779):
+        dst_mv = store.copy_model_version(src_mv, name2)
```

Comment:
Do we need to mock `time.time`? Can we do this instead?

```suggestion
    timestamp = time.time()
    assert copied_mv.creation_timestamp >= timestamp
```